components/kubeflow/kfserving/sample-pipeline.py

```diff
@@ -28,7 +28,11 @@ def kfservingPipeline(
     canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers',
```

Comment:
I would suggest removing this file. We will come up with a sample later in samples repo imho